content/docs/en/elements/components/progress.md

```diff
@@ -1,5 +1,19 @@
 ---
 title: Progress
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_progress_.progress
 ---
 
-TODO
+The Progress widget is a visual bar indicator of a progress in a operation. It shows a bar representing the current progress of the operation.
+
+---
+
+```html
+<Progress value="{{ someProperty }}" />
```

Comment:
Should use a vue binding: 

```html
<Progress :value="someProperty"/>
```

Reply:
Random questioon that I just thought about... would it be better to have an example for basic data binding, and a second example using the v-model directive instead ?